extensions/sql-database-projects/src/test/testContext.ts

```diff
@@ -57,6 +62,7 @@ export class MockDacFxService implements mssql.IDacFxService {
 	public getOptionsFromProfile(_: string): Thenable<mssql.DacFxOptionsResult> { return Promise.resolve(mockDacFxOptionsResult); }
 	public validateStreamingJob(_: string, __: string): Thenable<mssql.ValidateStreamingJobResult> { return Promise.resolve(mockDacFxResult); }
 	public parseTSqlScript(_: string, __: string): Thenable<mssql.ParseTSqlScriptResult> { return Promise.resolve({ containsCreateTableStatement: true }); }
+	public savePublishProfile(_: string, __: string, ___: string, ______?: Record<string, string>): Thenable<azdata.ResultStatus> { return Promise.resolve(mockSavePublishResult); }
```

Comment:
@Charles-Gagnon recommended changing the `___` variables to have useful names in another PR for performance + discard reasons.  I can't find it at this moment, but perhaps he can drop his knowledge again? :)

Reply:
these were the comments:
https://github.com/microsoft/azuredatastudio/pull/22452#discussion_r1149604509
https://github.com/microsoft/azuredatastudio/pull/22452#discussion_r1149608598

Reply:
Updated all the other methods as well :).